etc/uams/uams_dhx2_passwd.c

```diff
@@ -11,34 +11,32 @@
 
 #ifdef UAM_DHX2
 
-#include <atalk/standards.h>
-
+#include <arpa/inet.h>
+#include <errno.h>
+#include <pwd.h>
 #include <stdio.h>
 #include <stdlib.h>
 #include <string.h>
-#include <errno.h>
-#include <pwd.h>
-#include <arpa/inet.h>
+#include <sys/time.h>
+#include <time.h>
 #include <unistd.h>
 
 #ifdef HAVE_CRYPT_H
 #include <crypt.h>
 #endif
 
-#include <sys/time.h>
-#include <time.h>
+#ifdef HAVE_LIBGCRYPT
```

Comment:
FWIW libgcrypt is a mandatory dependency so this macro is arguably redundant.

Reply:
Best done in a separate commit, I think.

Reply:
Fair enough